libs/architectures/mlpe/architectures/embeddings/resnet.py

```diff
@@ -302,6 +302,10 @@ def __init__(
         norm_groups: Optional[int] = None,
     ) -> None:
         super().__init__()
+
+        # hack since other embedding require both n_ifos and strain
```

Comment:
Could you explain this to me

Reply:
Yeah so the embeddings require you pass `n_ifos` and `strain_dim`, but the `resnet` we've built determines this on its own. 

So, to make instantiation of the embedding in the `trainer` lib the same for all of the embeddings, I include this hack.